runtime/runtime/src/lib.rs

```diff
@@ -1261,6 +1241,9 @@ impl Runtime {
                 set_postponed_receipt(&mut state_update, &receipt);
             }
         }
+        if delayed_receipts_indices != DelayedReceiptIndices::default() {
```

Comment:
This looks a bit hacky. I, as a reader, am a bit confused if this can collide with a valid value or not. I would prefer Option::None to clearly express the intent.

Reply:
It follows the logic from `::apply`. It always pulls `delayed_receipts_indices` from the storage, but makes it a `default` if it's empty.

Reply:
I think it is okay, because `delayed_receipts_indices` is essentially two integer indices. In some cases it is okay to not wrap integer index into `Option` and instead rely on the default value.

Reply:
Relying on the default value like "zero" is C-style of implicit encoding some extra meaning in addition to the original one. Also, you need to look up the declaration to know that those are just two integers. Anyway, this is not a blocker, but rather a style concern (to me this makes the code harder to reason about).

Reply:
Yes, I think we are entering a territory of C vs Java style.